domain/lib/src/model/sharedspace/shared_space_role_name.dart

```diff
@@ -33,3 +33,9 @@
 enum SharedSpaceRoleName {
   READER, CONTRIBUTOR, WRITER, ADMIN
 }
+
+extension SharedSpaceRoleNameExtension on SharedSpaceRoleName {
+  String getEnumName() {
+    return toString().split('.').last.toLowerCase();
```

Comment:
It is for presentation? If it is, please move this to presentation and get the name from the `app_localization`, we need to translate it to depend on the language which user selected